ProteoformSuiteGUI/ExperimentExperimentComparison.cs

```diff
@@ -20,8 +20,8 @@ public ExperimentExperimentComparison()
         {
             InitializeComponent();
             this.dgv_EE_Peaks.MouseClick += new MouseEventHandler(dgv_EE_Peak_List_CellClick);
-            this.ct_EE_Histogram.MouseMove += new MouseEventHandler(ct_EE_Histogram_MouseMove);
```

Comment:
Are these MouseMove events used for the tool tip? If so, we probably shouldn't get rid of them.